packages/react-ui-map/src/Mapbox/LayerInteractions.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export default class LayerInteractions extends Component {
+  static propTypes = {
+    defaultMarkerId: PropTypes.string,
+    activeMarkerId: PropTypes.string,
+    children: PropTypes.node
+  }
+
+  static contextTypes = {
+    map: PropTypes.object
+  }
+
+  componentDidMount() {
+    const { map } = this.context
+    const { defaultMarkerId, activeMarkerId } = this.props
+
+    map.on('click', defaultMarkerId, (e) => {
+      map.flyTo({
```

Comment:
you may want to make this a separate method, within your component. Such as:
```javascript
map.on('click', defaultMarkerId, moveToMarker)
```
and then:
```javascript
moveToMarker(event) {
  const { map } = this.context
  const { activeMarkerId } = this.props

  map.flyTo({
    center: [
      event.lngLat.lng,
      event.lngLat.lat,
    ],
  })

  map.setFilter(activeMarkerId, ['==', 'title', event.features[0].properties.title])
}
```
This would give more context to what is going on, when someone is clicking the marker. Just an idea. If you go that route, make sure to add the method to the `constructor()`.